src/averageAges.js

```diff
@@ -15,11 +15,10 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  return people.filter(person => person.sex === 'm'
+  && (century === undefined || century === Math.ceil(person.died / 100)))
+    .map(men => men.died - men.born)
+    .reduce((acc, cur, index, arr) => acc + cur / arr.length, 0);
```

Comment:
Якщо не використовуєш індекс, можна його не писати, а просто поставити в тому місці кому
```suggestion
    .reduce((acc, cur, , arr) => acc + cur / arr.length, 0);
```

Reply:
не можу залишити пусте місце замість index, бо лінтер ругається і не пускає :(